libtrellis/CMakeLists.txt

```diff
@@ -143,3 +143,7 @@ else()
 endif()
 install(DIRECTORY ../database DESTINATION share/trellis PATTERN ".git" EXCLUDE)
 install(DIRECTORY ../misc DESTINATION share/trellis)
+install(DIRECTORY ../util/common DESTINATION share/trellis/util)
+install(DIRECTORY ../timing/util DESTINATION share/trellis/timing)
+install(DIRECTORY ../timing/util DESTINATION share/trellis/timing)
```

Comment:
Is timing/util supposed to appear twice?

Reply:
Of course not. Fixed it.